server/asset/dcr/live_test.go

```diff
@@ -68,14 +60,8 @@ func TestMain(m *testing.M) {
 			logger.Infof("done.")
 		}()
 
-		cfg := new(dcrdConfig)
-		err := flags.NewIniParser(flags.NewParser(cfg, flags.Default|flags.IgnoreUnknown)).ParseFile(dcrdConfigPath)
-		if err != nil {
-			fmt.Printf("error reading dcrd config: %v\n", err)
-			return 1
-		}
```

Comment:
I think all of this was a relic since `NewBackend` presently looks in the default location for the config file where these can be set and these `dcrdConfig` parameters are unused.